.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,28 @@
+---
+name: 🚀 Feature Request
+about: Suggest an idea for this project
+labels: enhancement
+---
+
+<!-- Please make sure you are posting a feature request related to Gridcoin. --> 
+
+<!-- For general questions about Gridcoin or wallet recovery please use one of the various communities:
+* Gridcoin on reddit https://www.reddit.com/r/gridcoin/
+* Discord https://discord.gg/jf9XX4a -->
+
+# Feature Request
+
+**Describe the Feature Request**
+<!-- A clear and concise description of what the feature request is. Please include if your feature request is related to a problem. -->
+
+**Describe Preferred Solution**
+<!-- A clear and concise description of what you want to happen. -->
+
+**Describe Alternatives**
+<!-- A clear and concise description of any alternative solutions or features you have considered. -->
+
+**Related Code**
+<!-- If you are able to illustrate the feature request with an example, please provide a samples via an online code collaborator such as [StackBlitz](https://stackblitz.com), or code snippet on [GitHub](https://github.com). -->
+
+**Additional Context**
+<!-- List any other information that is relevant to your issue. Stack traces, related issues, suggestions on how to add, use case, Stack Overflow links, forum links, screenshots, OS if applicable, etc. -->
```

Comment:
relevant to your feature request (not issue)

Reply:
Good catch! I also removed the `Stack traces` probably not relevant for a feature request.